README.md

```diff
@@ -39,6 +39,15 @@ The [Excel Custom Functions](https://github.com/OfficeDev/Excel-Custom-Functions
 * [office-addin-node-debugger](packages/office-addin-node-debugger/README.md)
 
   This package allows a Node instance to serve as a proxy for debugging a JavaScript runtime hosted by an Office application. 
+  
+* [office-addin-test-helpers](packages/office-addin-test-helpers/README.md)
+
+  This package provides test helpers for automating validation of Office Addins. Use in conjunction with the office-addin-test-server package and Mocha tests.
```

Comment:
This isn't dependent upon the test framework (Mocha). The wording could be revisted for clarity. @mscharlock @davidchesnut would appreciate your feedback.

Reply:
```suggestion
  This package provides tools that make validating your Office Add-in easier. You can use it with the office-addin-test-server package and the Mocha test framework (or another testing framework of your choice). 
```

Reply:
incorporated Michelle's suggested wording